packages/target-electron/src/deltachat/webxdc.ts

```diff
@@ -493,7 +493,7 @@ If you think that's a bug and you need that permission, then please open an issu
           accountId,
           msgId,
           update,
-          description
+          ''
         )
```

Comment:
It's a bit strange to set the parameter as deprecated, while at the same time we have to pass it to core to avoid an error there...

Should at least be an optional parameter in the type definitions in jsonrpc

Reply:
> Should at least be an optional parameter in the type definitions in jsonrpc

maybe jsonrpc was forgotten to be updated, again.

Reply:
Maybe I add this Option, since I'm adapting jsonrpc right now anyway, since it still misses `dc_msg_get_webxdc_href()` which is needed for https://github.com/deltachat/deltachat-desktop/issues/4366